component/src/main/java/org/wso2/extension/siddhi/gpl/execution/streamingml/classification/hoeffdingtree/util/AdaptiveHoeffdingModelsHolder.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
```

Comment:
Fix the license. Don't use apache license (use the GPL license).

Reply:
Updated license in [#c53fb67d8ae9e8faea253b9e0b16012a8f657fa0](https://github.com/wso2-extensions/siddhi-gpl-execution-streamingml/pull/16/commits/c53fb67d8ae9e8faea253b9e0b16012a8f657fa0)